exercises/bloco_7/dia_1/Atividade /Aula ao vivo.js

```diff
@@ -0,0 +1,28 @@
+function testingScope(escopo) {
+  if (escopo === true) {
+    var ifScope = 'Não devo ser utilizada fora do meu escopo (if)';
+    ifScope = ifScope + ' ótimo, fui utilizada no escopo !';
```

Comment:
O que você acha de remover o espaço entre a palavra e a exclamação? :p
```suggestion
    ifScope = ifScope + ' ótimo, fui utilizada no escopo!';
```